src/module.c

```diff
@@ -21,13 +21,16 @@
 #include "procedures/procedure.h"
 #include "arithmetic/arithmetic_expression.h"
 #include "graph/serializers/graphcontext_type.h"
+#include "graph/serializers/graphmeta_type.h"
 #include "redisearch_api.h"
 
 //------------------------------------------------------------------------------
 // Module-level global variables
 //------------------------------------------------------------------------------
-GraphContext **graphs_in_keyspace; // Global array tracking all extant GraphContexts.
-bool process_is_child;             // Flag indicating whether the running process is a child.
+GraphContext **graphs_in_keyspace;  // Global array tracking all extant GraphContexts.
+bool process_is_child;              // Flag indicating whether the running process is a child.
+uint64_t entities_threshold;   		// The limit of number of entities encoded at once.
+uint redis_major_version;			// The redis server major version.
```

Comment:
Use spaces instead of tabs for non-initial whitespace.

Also, make `entities_is_threshold` a `long long` or alter casting in config.c